homeassistant/components/onewire/sensor.py

```diff
@@ -30,6 +30,7 @@
     "28": {"temperature": "temperature"},
     "3B": {"temperature": "temperature"},
     "42": {"temperature": "temperature"},
+    "7E": {"temperature": "EDS0068/temperature", "pressure": "EDS0068/pressure"},
```

Comment:
We shouldn't add more features to this platform until the decide interface, I/O and data parsing has been extracted to a standalone library published on pypi.

https://developers.home-assistant.io/docs/en/creating_platform_code_review.html#4-communication-with-devices-services